client/view.html

```diff
@@ -30,7 +30,6 @@
 						<th>Player Name</th>
 						<th>Cards</th>
 						<th>Current actor</th>
-						<th>Game Owner</th>

```

Comment:
did you mean to remove this? the TD.js-owner still exists

Reply:
Yeah I meant to remove this, it's not needed is it? I put it in to visualise who owned the game.

There is a column on the end for where this is/was, but only the game owner can see it in their view as it's where the dealer button form is. 

With with this left in, it just looks odd and empty on other views.